ui_framework/src/components/popover/_popover.scss

```diff
@@ -1,7 +1,8 @@
 // Pop menu is an animated popover relatively positioned to a button / action.
 // By default it positions in the middle, but can be anchored left and right.
 
-@include component('kuiPopMenu') {
+@include component('kuiPopover') {
```

Comment:
The problem we need to solve is that the original way the Popover was implemented caused the arrow to overlap the content, which works great for the Header, but not so great for anything else. You can see this on the Popover example page:

![image](https://user-images.githubusercontent.com/1238659/28590813-5b040478-7138-11e7-80c9-b3c736045dd1.png)